setup.py

```diff
@@ -11,6 +11,7 @@ def parse_requirements(requirement_file):
 
 
 setup(
+    version="4.0.0",
```

Comment:
Should make sure everyone knows this is now manually versioned independently of the git tag, travis will still expect a tag to cut release.

Reply:
What do you suggest?

Reply:
We should still go through the same process with Travis for this release and make sure we cut the release with the same version tag so that everything will work as we expect, then come back to the build process after this sprint.